ckanext/datadotworld/api.py

```diff
@@ -121,6 +121,7 @@ class API:
     api_res_delete = api_res_create + '/{file}'
 
     auth = 'Bearer {key}'
+    user_agent_header = 'ckanext-datadotworld'
```

Comment:
Could you please include version, as well? Dynamically pulled from the python package, if possible.